docs/blog/newsletter-june-2022.rst

```diff
@@ -0,0 +1,129 @@
+
+A longer read: Now a national holiday in the United States, June 19th is Juneteenth. Want to learn more about it? Oprah `explains the history of Juneteenth <https://www.oprahdaily.com/life/a32893726/what-is-juneteenth/>`__.
+
+----------------
+From our sponsor
+----------------
+
+This month’s newsletter is sponsored by SPONSOR:
+
+.. raw:: html
+
+    <hr>
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
+      <tbody>
+        <tr>
+          <td width="75%">
+              <p>
+              CONTENT
+              </p>
+          </td>
+          <td width="25%">
+            <a href="https://www.LINK.COM">
+              <img style="margin-left: 15px;" alt="SPONSOR" src="/_static/img/sponsors/rtd.png">
+            </a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <hr>
+
+*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
+
+------------------
+Featured job posts
+------------------
+
+- `Technical Writer <https://jobs.writethedocs.org/job/718/technical-writer/>`__, Wix (Vilnius, Lithuania )
+- `Technical Writer (Developer Documentation) <https://jobs.writethedocs.org/job/717/technical-writer-developer-documentation-fully-remote/>`__,  Humanitec GmbH (Remote)
+-  `Technical Writer, Serverless, Google Cloud <https://jobs.writethedocs.org/job/731/technical-writer-serverless-google-cloud/>`__, Google (Kitchener, ON, Canada)
+- `Technical Writer (IntelliJ) <https://jobs.writethedocs.org/job/735/technical-writer-intellij/>`__, JetBrains (arious locations)
+- `Senior Manager, Technical Writing, Developer Acceleration Platforms <https://jobs.writethedocs.org/job/729/senior-manager-technical-writing-developer-acceleration-platforms/>`__, Google (Seattle, WA, USA)
+- `Technical Content Strategist, Developer Relations Content, Google Cloud <https://jobs.writethedocs.org/job/727/technical-content-strategist-developer-relations-content-google-cloud/>`__, Google (Seattle, WA, USA)
+- `Product Documentation Manager <https://jobs.writethedocs.org/job/714/product-documentation-manager/>`__, ReadMe (Remote)
+- `Technical Writer, Software Engineering <https://jobs.writethedocs.org/job/733/technical-writer-software-engineering/>`__, Google (Remote)
+- `Developer Educator - Remote APJ <https://jobs.writethedocs.org/job/725/developer-educator-remote-apj/>`__, Twilio (Remote)
+- `Developer Evangelist <https://jobs.writethedocs.org/job/726/developer-evangelist/>`__, Twilio (Remote)
+
+*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
+
+------------------------
+Virtual events coming up
+------------------------
+
+- 02 June, 19:00 PDT (Los Angelos) - `June: Catch up and discussion <https://www.meetup.com/Write-the-Docs-LA/events/285814451/>`__
```

Comment:
Depending on when we ship, maybe remove this line?

I think there's a Florida morning social coming up too, although I don't know how exhaustive we're being with this list.

Reply:
I think the one in Florida is the same -- an East Coast quorum. I'll try to make that clear